packages/language-service/test/lib/diagnostics.spec.ts

```diff
@@ -169,5 +170,39 @@ describe('diagnostics', () => {
 
             expect(diagnostics).to.eql([]);
         });
+
+        it('should ignore errors for attribute selectors inside @st-scope', () => {
+            const filePath = '/style.st.css';
```

Comment:
Unify the two tests, you can test both cases at once

Reply:
done